lib/ansible/modules/apt.py

```diff
@@ -236,9 +236,9 @@
 
 - name: Install a list of packages
   ansible.builtin.apt:
-    pkg:
```

Comment:
`pkg` is still functional alias to `name`, this is to let people know that it still works, as documented.